packages/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -52,7 +52,8 @@ export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
   const settings = new LocalStorageStore<MarkdownSettingsProps>(MARKDOWN_PLUGIN, {
     defaultViewMode: 'preview',
     toolbar: true,
-    folding: false,
+    numberedHeadings: true,
+    folding: true,
```

Comment:
I don't think this behaves well enough to be on by default

Reply:
example?

Reply:
- they don't sit next to the text so if the plank is wider than max text width they look broken
- they seem to render after the text does and shift it over on first load
- there was also the z-index issue with them, but I think that one was fixed?

Reply:
reverted since we're trying to make the initial experience fast and clean.